xml/cha_introduction.xml

```diff
@@ -94,16 +94,13 @@
      &gc;
     </para>
    </listitem>
-   <listitem>
-    <para>
-     &oc;
-    </para>
-   </listitem>
```

Comment:
We don't do Oracle? 
They are listed both at https://www.suse.com/solutions/public-cloud/ and https://www.suse.com/support/public-cloud-support/

Reply:
Ok, seems the next paragraph answers that question.